src/application/tests/testResponseHeader.ts

```diff
@@ -9,18 +9,26 @@ export const testResponseHeader = (
   _aOperation: OasMappedOperation | null,
   config?: GlobalConfig
 ): PostmanMappedOperation => {
-  // Separator
-  const split = config?.separatorSymbol ?? '::'
-  // Check - Response header check
-  const pmTest: string = [
-    `// Validate if response header is present \n`,
-    `pm.test("[${pmOperation.method.toUpperCase()}]${split}${pmOperation.path}`,
-    ` - Response header ${headerName} is present", function () {\n`,
-    `   pm.response.to.have.header("${headerName}");\n`,
-    `});\n`
-  ].join('')
+  // Get header from OpenAPI operation
+  const headers = _aOperation?.requestHeaders
```

Comment:
We need to check the response headers, not the request headers

Reply:
Very good catch, totally missed that.